File: java/src/test/java/com/testexercise/ebayframework/ItemPage.java
package com.testexercise.ebayframework;

import org.openqa.selenium.*;

public class ItemPage extends BasePage
{

    public ItemPage(AutomationFramework af)
    {
        super(af);
    }
    
    private final String addToCartId = "isCartBtn_btn";
    private final String itemTitleId = "itemTitle";
    private final String itemPriceId = "prcIsum";
    
    public void addItemtoCart()
    {
        af.driver().findElement(By.id(addToCartId)).click();
    }
    
    public String getItemTitle()
    {
        return af.driver().findElement(By.id(itemTitleId)).getText();
    }
    
    public String getItemPrice()
    {
        return af.driver().findElement(By.id(itemPriceId)).getText();
    }
}

File: net/TestExerciseEbay/EbayPOMFramework/ItemPage.cs
using System;
using OpenQA.Selenium;

namespace TestExercise.EbayPOMFramework
{
    public class ItemPage : BasePage
    {

        public ItemPage(AutomationFramework af) : base(af) { }

        private const string addToCartId = "isCartBtn_btn";
        private const string itemTitleId = "itemTitle";
        private const string itemPriceId = "prcIsum";

        public void AddItemtoCart()
        {
            af.Driver.FindElement(By.Id(addToCartId)).Click();
        }

        public String ItemTitle => af.Driver.FindElement(By.Id(itemTitleId)).Text;

        public String ItemPrice => af.Driver.FindElement(By.Id(itemPriceId)).Text;
    }
}
